soccer/gameplay/constants.py

```diff
@@ -26,145 +26,4 @@ class Ball:
-    Field.GoalDepth = value.GoalDepth()
-    Field.GoalHeight = value.GoalHeight()
-    Field.PenaltyDist = value.PenaltyDist()
-    Field.PenaltyDiam = value.PenaltyDiam()
-    Field.ArcRadius = value.ArcRadius()
-    Field.CenterRadius = value.CenterRadius()
-    Field.CenterDiameter = value.CenterDiameter()
-    Field.GoalFlat = value.GoalFlat()
-    Field.FloorLength = value.FloorLength()
-    Field.FloorWidth = value.FloorWidth()
-
-    Field.CenterPoint = robocup.Point(0.0, Field.Length / 2.0)
-
-    Field.OurGoalZoneShape = robocup.CompositeShape()
-    Field.OurGoalZoneShape.add_shape(robocup.Circle(
-        robocup.Point(-Field.GoalFlat / 2.0, 0), Field.ArcRadius))
-    Field.OurGoalZoneShape.add_shape(robocup.Circle(
-        robocup.Point(Field.GoalFlat / 2.0, 0), Field.ArcRadius))
-    Field.OurGoalZoneShape.add_shape(robocup.Rect(
-        robocup.Point(-Field.GoalFlat / 2.0, Field.ArcRadius), robocup.Point(
-            Field.GoalFlat / 2.0, 0)))
-
-    Field.TheirGoalShape = robocup.CompositeShape()
-    Field.TheirGoalShape.add_shape(robocup.Circle(
-        robocup.Point(-Field.GoalFlat / 2.0, Field.Length), Field.ArcRadius))
-    Field.TheirGoalShape.add_shape(robocup.Circle(
-        robocup.Point(Field.GoalFlat / 2.0, Field.Length), Field.ArcRadius))
-    Field.TheirGoalShape.add_shape(robocup.Rect(
-        robocup.Point(-Field.GoalFlat / 2.0, Field.Length), robocup.Point(
-            Field.GoalFlat / 2.0, Field.Length - Field.ArcRadius)))
-
-    Field.TheirGoalSegment = robocup.Segment(
-        robocup.Point(Field.GoalWidth / 2.0, Field.Length),
-        robocup.Point(-Field.GoalWidth / 2.0, Field.Length))
-    Field.OurGoalSegment = robocup.Segment(
-        robocup.Point(Field.GoalWidth / 2.0, 0),
-        robocup.Point(-Field.GoalWidth / 2.0, 0))
-
-    Field.TheirHalf = robocup.Rect(
-        robocup.Point(-Field.Width / 2, Field.Length), robocup.Point(
-            Field.Width / 2, Field.Length / 2))
-    Field.OurHalf = robocup.Rect(
-        robocup.Point(-Field.Width / 2, 0), robocup.Point(Field.Width / 2,
-                                                          Field.Length / 2))
-
-    Field.FieldBorders = [
-        robocup.Line(
-            robocup.Point(-Field.Width / 2.0, 0),
-            robocup.Point(-Field.Width / 2.0, Field.Length)), robocup.Line(
-                robocup.Point(-Field.Width / 2.0, Field.Length), robocup.Point(
-                    Field.Width / 2.0, Field.Length)), robocup.Line(
-                        robocup.Point(Field.Width / 2.0, Field.Length),
-                        robocup.Point(Field.Width / 2.0, 0)), robocup.Line(
-                            robocup.Point(Field.Width / 2.0, 0), robocup.Point(
-                                -Field.Width / 2.0, 0))
-    ]
-
-    Field.FieldRect = robocup.Rect(
-        robocup.Point(-Field.Width / 2.0, 0), robocup.Point(Field.Width / 2.0,
-                                                            Field.Length))
+Field = None
```

Comment:
Can you add a comment above `Field = None` saying that it gets set to the global `Field_Dimensions` object by the `GameplayModule`?